invenio/utils/persistentid.py

```diff
@@ -371,6 +390,14 @@ def normalize_orcid(val):
     return "-".join([val[0:4], val[4:8], val[8:12], val[12:16]])
 
 
+def normalize_gnd(val):
+    """Normalize a GND identifier."""
+    if val.startswith("http://d-nb.info/gnd/"):
```

Comment:
Here you can assume you have a valid GND identifier. Current a valid GND can only have one form, so I'm about to think we don't need the normalise function